### ISSUE TITLE
Add storybook a11y addon

### ISSUE THREAD

**Issue by brandonlenz** [trussworks/unemployment-insurance-modernization-demo#17]
https://storybook.js.org/addons/@storybook/addon-a11y

AC:
As a storybook user, I can see the "Accessibility" tab and inspect my stories for a11y violations
